components/Control/Control.js

```diff
@@ -18,7 +19,9 @@ export default function Control({
           calculated={calculated}
         />
       )}
-      {calculated && !isLoading && <ControlButton job="save" />}
+      {calculated && !isLoading && (
+        <ControlButton handler={onSave} job="save" />
```

Comment:
to be conform with the conventions:

```suggestion
        <ControlButton onSave={onSave} job="save" />
```

Reply:
Morning Niklas, 
thanks for taking care about my PR.
I've implemented the other suggestions you made. But here I am reusing the ControlButton component three times and give every button its own handler. That's why I named it handler and not onSave. Maybe I could name it onClick. That would be universal and includes a "on"😁

Reply:
That's a great idea 👍